pkg/oc/cli/admin/release/extract.go

```diff
@@ -58,6 +78,9 @@ type ExtractOptions struct {
 
 	From string
 
+	// GitPath is the path of a root directory to extract the source of a release to.
+	GitPath string
```

Comment:
nit: TargetGitPath or RepositoriesCheckoutPath otherwise this will be confused with `/usr/bin/git`